mirascope/core/base/from_call_args.py

```diff
@@ -0,0 +1,23 @@
+class FromCallArgs:
+    """
+    If you want to pass the function arguments to the response model, you can use this metadata.
```

Comment:
nit: we need to follow google style docstrings for api generation, so there needs to be a single short line:

```
"""Use this metadata to pass function arguments through to the response model.

Example:

...
```